docs/how_to_use/how_to_build.md

```diff
@@ -0,0 +1,256 @@
+
+  ```shell
+  bazel build --config ${os} --config avx512_${os} --//:polygon_zkevm_backend //...
+  ```
+
+### Hardware acceleration
+
+#### CUDA backend
+
+- `--config cuda`: Enable [cuda] backend.
+
+  ```shell
+  bazel build --config ${os} --config cuda //...
+  ```
+
+#### ROCm backend
+
+- `--config rocm`: Enable [rocm] backend.
+
+  ```shell
+  bazel build --config ${os} --config rocm //...
+  ```
+
+_NOTE_: Although there is a `rocm` option, it is recommended not to use it at the moment because it is not being tested yet.
+
+[cuda]: https://developer.nvidia.com/cuda-toolkit
+[rocm]: https://www.amd.com/en/graphics/servers-solutions-rocm
+
+### .bazelrc.user
+
+You can set the options in `.bazelrc.user`.
+
+For example:
+
+```
+build --config linux
+build --config dbg
+build --config polygon_zkevm_backend
+```
+
+If you add `.bazelrc.user` like above, you can build with the options without specify in the command.
+
+```shell
+bazel build //...
+# It is same as:
+# bazel build --config linux --config dbg --config polygon_zkevm_backend //...
+```
+
+## Build from Bazel repository
+
+You can build Tachyon on your own bazel project. The following snippet is an example of `WORKSPACE` to use Tachyon in an external project.
+You can get the Tachyon repo in specific commit. You'll also need a sha256 value of the fetched code.
+
+```shell
+wget https://github.com/kroma-network/tachyon/archive/c6a38895cb6f9b9f73e3d64928d85a9762b66639.tar.gz
+
+shasum -a 256 c6a38895cb6f9b9f73e3d64928d85a9762b66639.tar.gz
+3ebc4edca153c9b93de200d5e22be66d1ff7ea52e2793c83947607c532dfacbc  c6a38895cb6f9b9f73e3d64928d85a9762b66639.tar.gz
+```
+
+```py
+// WORKSPACE 
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "kroma_network_tachyon",
+    sha256 = "3ebc4edca153c9b93de200d5e22be66d1ff7ea52e2793c83947607c532dfacbc",
+    strip_prefix = "tachyon-c6a38895cb6f9b9f73e3d64928d85a9762b66639",
+    urls = ["https://github.com/kroma-network/tachyon/archive/c6a38895cb6f9b9f73e3d64928d85a9762b66639.tar.gz"],
+)
+```
+
+## Debian packaging
+
+Build a Debian package with the supported scheme(only halo2 for now) and the [options](#options) you want.
+For example, to build the Halo2 Debian package with `polygon_zkevm_backend` and `has_openmp` options, run the following command:
+
+```shell
+bazel build -c opt --config halo2 --//:polygon_zkevm_backend --//:has_openmp  --//:c_shared_object //scripts/packages/debian/runtime:debian
+bazel build -c opt --config halo2 --//:polygon_zkevm_backend --//:has_openmp  --//:c_shared_object //scripts/packages/debian/dev:debian
+
+sudo dpkg -i bazel-bin/scripts/packages/debian/runtime/libtachyon_0.0.1_amd64.deb
+sudo dpkg -i bazel-bin/scripts/packages/debian/dev/libtachyon-dev_0.0.1_amd64.deb
+
+```
+
+## Frequently Asked Questions
+
+#### Debugging on macOS
```

Comment:
Did you turn on markdown lint? you have to increase the heading level one by one.